### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by EgorSeverin**

### CODE REVIEW

File: src/TeachMeSkills/Homework/Homework2.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            case 4 -> "Зелёный";
+            case 5 -> "Голубой";
+            case 6 -> "Синий";
+            case 7 -> "Фиолетовый";
+            default -> "Введённое число не является номером цвета радуги";
+        };
+        System.out.println(color);
+
+        // Циклы. Задание 1
+        for(number = 0; number < 100; number++) {
```
Comment (DHrytsuk) [inline src/TeachMeSkills/Homework/Homework2.java]: We could start from 1

File: src/TeachMeSkills/Homework/Homework2.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            if (number % 2 != 0) {
+                System.out.print(" " + number);
+            }
+        }
+
+        // Циклы. Задание 2
+        number = sc.nextInt();
+        int result = 0;
+        if (number >= 0) {
+            for (int sum = 2; sum < number; sum++) {
```
Comment (DHrytsuk) [inline src/TeachMeSkills/Homework/Homework2.java]: Is it correct?

File: src/TeachMeSkills/Homework/Homework2.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        i = 10;
+        while(i <= 20) {
+            System.out.print(" " + i * i);
+            i++;
+        }
+
+        // Доплнительно. Задание 1
+        System.out.print("Введите желаемое количество выводимых членов последовательности Фибоначчи: ");
+        int n = sc.nextInt();
+        int[] arr = new int[n];
```
Comment (DHrytsuk) [inline src/TeachMeSkills/Homework/Homework2.java]: We don't need arrays here

File: src/TeachMeSkills/Homework/Homework2.java (hunk 2 above)
Comment (EgorSeverin) [reply src/TeachMeSkills/Homework/Homework2.java]: Yes, if we exclude 1 and the entered number

File: src/TeachMeSkills/Homework/Homework2.java (hunk 3 above)
Comment (EgorSeverin) [reply src/TeachMeSkills/Homework/Homework2.java]: Сorrected

File: src/TeachMeSkills/Homework/Homework2.java (hunk 2 above)
Comment (DHrytsuk) [reply src/TeachMeSkills/Homework/Homework2.java]: But why did you exclude 1?

File: src/TeachMeSkills/Homework/Homework2.java [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+            i++;
+        }
+
+        // Доплнительно. Задание 1
+        System.out.print("\nВведите желаемое количество выводимых членов последовательности Фибоначчи: ");
+        int n = sc.nextInt();
+        int number1 = 0;
+        int number2 = 1;
+        int number3;
+        System.out.print(number1 + " " + number2 + " ");
```
Comment (DHrytsuk) [inline src/TeachMeSkills/Homework/Homework2.java]: What if n = 1?
Is this output will be correct?

Comment (EgorSeverin) [reply src/TeachMeSkills/Homework/Homework2.java]: Corrected

File: src/TeachMeSkills/Homework/Homework2.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+        // Доплнительно. Задание 1
+        System.out.print("\nВведите желаемое количество выводимых членов последовательности Фибоначчи: ");
+        int n = sc.nextInt();
+        int number1 = 0;
+        int number2 = 1;
+        int number3;
+        System.out.print(number1 + " ");
+        if(n > 1) {
+            System.out.print(number2 + " ");
```
Comment (DHrytsuk) [inline src/TeachMeSkills/Homework/Homework2.java]: We should have only one sout and place in inside the loop

File: src/TeachMeSkills/Homework/Homework2.java (hunk 2 above)
Comment (DHrytsuk) [reply src/TeachMeSkills/Homework/Homework2.java]: Not fixed

Comment (EgorSeverin) [reply src/TeachMeSkills/Homework/Homework2.java]: fixed

**Review by DHrytsuk — APPROVED** [review]

**Pull request merged**